corehq/apps/commtrack/tests/test_fixture.py

```diff
@@ -1,20 +1,36 @@
+import datetime
 import random
 import string
 from xml.etree import ElementTree
 
+from django.test import TestCase
+
 from casexml.apps.case.xml import V1
+from casexml.apps.phone.models import SyncLog
 from casexml.apps.phone.tests.utils import generate_restore_payload, create_restore_user
 from corehq.apps.app_manager.tests.util import TestXmlMixin
 from corehq.apps.programs.fixtures import program_fixture_generator
 from corehq.apps.products.fixtures import product_fixture_generator
 from corehq.apps.products.models import Product
 from corehq.apps.programs.models import Program
-from corehq.apps.commtrack.tests.util import CommTrackTest
-from casexml.apps.phone.models import SyncLog
-import datetime
+from corehq.apps.commtrack.tests import util
+
+
+class FixtureTest(TestCase, TestXmlMixin):
+    domain = "fixture-test"
 
+    @classmethod
+    def setUpClass(cls):
+        super(FixtureTest, cls).setUpClass()
+        cls.domain_obj = util.bootstrap_domain(cls.domain)
+        util.bootstrap_location_types(cls.domain)
+        util.bootstrap_products(cls.domain)
+        cls.user = create_restore_user(cls.domain)
 
-class FixtureTest(CommTrackTest, TestXmlMixin):
+    @classmethod
+    def tearDownClass(cls):
+        cls.domain_obj.delete()
```

Comment:
I don't think so - `create_restore_user` creates a restore user instance, which isn't a thing that can be saved to the db, and a CommCareUser, which is directly tied to a domain, meaning `domain_obj.delete()` takes care of cleaning it up.